website/source/layouts/docs.erb

```diff
@@ -141,6 +141,19 @@
 
 					<li<%= sidebar_current("docs-commands-operator") %>>
 					<a href="/docs/commands/operator.html">operator</a>
+					<ul class="subnav">
```

Comment:
I'd say get rid of `list-peers` and `remove-peers` so it looks like this:

* Consul Commands (CLI)
    * operator
        * raft